### ISSUE TITLE
craft.formie.submissions is loading form from Trashed first if sharing the same handle

### ISSUE THREAD

**Issue by romainpoirier** [verbb/formie#715]
**Description**

I have two forms, and each one has submissions. I can try to print their submissions using `{{ craft.formie.submissions.form(handle).one() }}`. The first one print results on front-end, the second one not, while both have in the CP complete submissions in the same site group.

For the other one without printed submission, I found that on front-end Formie was loading the wrong form. In fact, after importing errors, I had to delete my forms several times. Formie was trying to load the one in `Trashed` before the active one in `All`.

Both were using the same handle, but for a strange reason that was the one in `Trashed` that was trying to load on front-end instead of the active one.

### Form settings
- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.5.5
- Craft version: 3.7.28
- Multi-site: Yes

**Comment by engram-design** [comment]
So when you delete a form, it should delete all the submissions along with it. Can you confirm if both your submissions are un-trashed, because the on related to the deleted form should also be deleted.

I think this should be fixed for the next release though. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.5",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.6](https://github.com/verbb/formie/releases/tag/1.5.6)
